cgi-bin/LJ/Talk.pm

```diff
@@ -1599,7 +1599,8 @@ sub talkform {
 
             $ret .= " " . $BML::ML{'.opt.willscreen'} if $screening eq 'A'
                     || ( $screening eq 'R' && !$remote->is_validated )
-                    || ( $screening eq 'F' && !$journalu->trusts($remote) ) ;
+                    || ( $screening eq 'F' && !$journalu->trusts($remote) )
+                    || ( $journalu->has_autoscreen( $remote ) );
```

Comment:
...Right. It looks mismatched to me without them, but i'm also a somewhat overcautious parenthesizer. I'll take them out.